Makefile

```
@@ -41,6 +41,11 @@ fast_tests_diffusers:
 	python -m pip install .[tests]
 	python -m pytest tests/test_diffusers.py
 
+# Run unit tests related to ClipSeg
+fast_tests_clipseg:
+	python -m pip install .[tests]
+	python -m pytest tests/test_clipseg.py
```

```suggestion
slow_tests_object_segmentation: test_installs
	python -m pytest tests/test_clipseg.py
```
Let's rather consider it as a slow test command as it contains a regression test and keep fast tests for generic stuff.

---

Sure, that makes sense.